src/HaskellerAnswers.hs

```diff
@@ -27,14 +30,20 @@ runHaskellerAnswers = do
         }
 
 data Model = Model
-    { modelAllAnswers    :: !(NonEmpty Text)
+    { modelAllAnswers    :: ![Text]
     , modelCurrentAnswer :: !Text
+    , modelRandGen       :: !StdGen
     } deriving stock (Eq, Show)
 
-defaultModel :: Model
-defaultModel = Model
-    { modelAllAnswers    = answers
+-- orphan instance because StdGen doesn't have Eq
+instance Eq StdGen where
+    gen1 == gen2 = show gen1 == show gen2
+
+mkDefaultModel :: StdGen -> Model
+mkDefaultModel gen = Model
+    { modelAllAnswers    = defaultAnswers
```

Comment:
Should we shuffle them here as well?

Reply:
No, I don't think that we need to shuffle, because each time this `Model` is created there will be a new `StdGen`, so each time a new answer will be chosen.

Reply:
Yes, makes sense :+1: